content/direction/enablement/index.md

```diff
@@ -10,35 +10,39 @@ Our mission is to provide the right technical foundations critical to the succes
+We are focused on several themes to help make our one-year vision a reality. They are not listed in priority order, as we are pursuing all of them. To see how these play into what any particular product area is delivering, take a look at the individual team direction pages, as indexed on our [direction home page](../index.md#enablement)
 
 ### Make it easier to contribute to our frontend codebase
 
 Developing a component system leads to a consistent user experience, faster delivery, and more inclusive design. To ensure better adoption and customer satisfaction with our product we are also focused on improving page rendering performance and perceived performance when a user interacts with the application.
 
 ### Reduce friction in deployment and upgrade
 
-Enable Sourcegraph customers and prospects to easily access the value our product provides without tripping over blocks in the install & upgrade process. Ensure Sourcegraph can be run by any user with a standard engineering background at scale and with ease. Set clear expectations with customers / prospects on the environment requirements necessary to manage Sourcegraph at the scale they seek.
+Enable Sourcegraph customers and prospects to easily access the value our product provides without experiencing friction during the install & upgrade process. Ensure Sourcegraph can be run by any user with a standard engineering background at scale and with ease. Set clear expectations with customers / prospects on the environment requirements necessary to manage Sourcegraph at the scale they seek.
 
-### Delight our site admins
+### Strengthen support of code management systems
 
-Site admin’s are a critical part of our success. We want to ensure configuring sourcegraph is intuitive and does not create unnecessary burden in their process.
+As we broaden our reach, prospects and customers look to us to support a large variety of code management systems, in a way that is robust and scalable. We want to provide our user base the means to simply, easily, and reliably connect to all common code management systems.
 
-### Expand support of code management systems
+### Enrich education and documentation systems
 
-As we broaden our reach, prospects and customers look to us to support a large variety of code management systems. We want to provide our user base the means to simply, easily, and reliably connect to all common code management systems.
+Our company as well as user base is growing rapidly. We want to see all people in our ecosystem succeed and to do so we need enriched education and documentation systems.
+
+### Improve our continuous integration pipeline
+
+When our CI pipeline breaks we need clearer indications of what is causing the failures so that we can quickly take action on resolving the issues.
```

Comment:
```suggestion
When our CI pipeline breaks we need clear indications and rich observability of what is causing the failures so that we can quickly take action on resolving the issues.
```